plenum/persistence/req_id_to_txn.py

```diff
@@ -0,0 +1,50 @@
+from hashlib import sha256
+from typing import Optional
+
+from plenum.persistence.kv_store import KVStoreLeveldb
+
+
+class ReqIdrToTxn:
+    """
+    Stores a map from client identifier, request id tuple to transaction
+    sequence number
+    """
+    def __init__(self, *args, **kwargs):
+        raise NotImplementedError
+
+    def add(self, identifier, reqId, seqNo):
+        raise NotImplementedError
+
+    def get(self, identifier, reqId):
+        raise NotImplementedError
+
+
+class ReqIdrToTxnLevelDB(ReqIdrToTxn):
```

Comment:
What if we add another implementation (multiple implementations) for this interface? I'm ok with having interface and a simple implementation in one file if we're sure that this is the only implementation in the scope of this project (some test or default one).
If we can see multiple implementations in one project from the very beginning (like leveldb, rocksdb, redis), then I would put them into separate files.